src/share/recipes/install/bitwarden.sh

```diff
@@ -45,3 +79,45 @@ Completed. You can use Bitwarden Desktop App instantly, like:
 Enjoy! ;)
 
 EOS
+
+}
+
+doit() {
+    log info "** Installing bitwarden:"
+    download_appimage
+    check_hashsum
+    install_app_image
+    setup_gnome_assets
+    inform_user
+}
+
+undo() {
+    log info "** Un-installing bitwarden:"
+
+    result=''
+
+    # Remove gnome assets
+    log info "Removing gnome assets..."
+    rm -f "${_LOCAL}/opt/bitwarden/bitwarden-icon.png"
+    result="${result}:$?"
+    rm -f "${_LOCAL}/share/applications/com.bitwarden.desktop"
+    result="${result}:$?"
+
+    # Remove symlink
+    log info "Removing symlink..."
+    rm -f "$_LOCAL/bin/bitwarden-desktop"
+    result="${result}:$?"
+
+    # Remove app image file
+    log info "Removing bitwarden app image..."
+    rm -f "${app_fullpath}"
+    result="${result}:$?"
+
+    if [ $result = ':0:0:0:0' ]; then
```

Comment:
This would fail if $result is empty - please use 'x'! ;)

Reply:
Indeed! Fixed with 703d30c.